src/models/volunteer.js

```diff
@@ -0,0 +1,58 @@
+const mongoose = require('mongoose');
+
+const volunteer = new mongoose.Schema(
+  {
+    firstName: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    password: {
+      type: String,
+      required: true,
```

Comment:
`password` won't be required always, because we might be social network auth too, like google auth.